src/main/java/hudson/remoting/UserRequest.java

```diff
@@ -86,10 +86,32 @@ public UserRequest(Channel local, Callable<?,EXC> c) throws IOException {
         return result;
     }
 
+    static boolean workaroundDone = false;
     protected UserResponse<RSP,EXC> perform(Channel channel) throws EXC {
         try {
             ClassLoader cl = channel.importedClassLoaders.get(classLoaderProxy);
 
+            // Allow forcibly load of a class, allows to workaround:
+            // @See        https://issues.jenkins-ci.org/browse/JENKINS-19445
+            // @Related    https://issues.tmatesoft.com/issue/SGT-451
+            final String clazz = System.getProperty(RemoteClassLoader.class.getName() + ".force", null);
```

Comment:
This is the only point we are certain to have the same ClassLoader with the same rules. Not sure how to obtain the same ClassLoader from a static block. We checked all the flow from Launcher to here, as the only point were it would be safe, and early enough to prevent the blockage.